steppy/base.py

```diff
@@ -540,13 +568,13 @@ def __str__(self):
 
 
 class BaseTransformer:
-    """Abstraction on two level fit and transform execution.
+    """Abstraction on ``fit`` and ``transform`` execution.
 
     Base transformer is an abstraction strongly inspired by the ``sklearn.Transformer`` and
     ``sklearn.Estimator``. Two main concepts are:
 
         1. Every action that can be performed on data (transformation, model training) can be
-        performed in two steps: fitting (where trainable parameters are estimated) and transforming
+        performed in two steps: fitting (where fittable parameters are estimated) and transforming
```

Comment:
Actually trainable/non-trainable parameters is used in many places (tensorflow/keras/pytorch) wheras I am not sure that fittable parameters exists.